### DISCUSSION TITLE
DCPError when changing the power of noise in communication problems

### DISCUSSION THREAD

**Discussion by Fengcheng-Pei** [cvxpy/cvxpy#2003] [Q&A]
Hello,

When I use cvxpy to deal with Signal-to-Noise related problems, DCPErrors will occur when I change the noise power from 10^(-6) to 10^(-11). I really do not know why, because it works well when the noise power equals to 10^(-6). Could it be that cvxpy could not process this extremely small number?

Best regards,
Fengcheng Pei

**Comment by phschiele** [comment]
Hi @Fengcheng-Pei, 
can you share which atom you are plugging these values into?
In any case, 10^-11 is almost indistinguishable from 0 for a computer. So even if you're model still makes sense mathematically, it is possible that it will no longer work with CVXPY (or other software as well). Often there are ways to rescale the problem. 

It would help if you could share a minimum working example!
